student.rb

```diff
@@ -6,6 +6,11 @@ def initialize(classroom, id, age, name: 'Unknown', parent_permission: true)
     @classroom = classroom
   end
 
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.includes?(self)
```

Comment:
- [ ]  Please as mentioned also provide ``attr_reader :classroom`` when you use attr_reader, it only creates a getter method So You can use classroom to retrieve the value of the @classroom instance variable.